packages/cozy-harvest-lib/src/components/AccountForm.jsx

```diff
@@ -105,20 +105,23 @@ export class AccountForm extends PureComponent {
   }
 
   render() {
-    const { account, fields, oauth, t } = this.props
+    const { fields, initialValues, oauth, t } = this.props
 
     if (oauth) return <OAuthForm oauth={oauth} />
 
     const sanitizedFields = Manifest.sanitizeFields(fields)
-    const initialValues = account ? account.auth : {}
     return (
       <Form
         initialValues={initialValues}
         // eslint-disable-next-line no-console
         onSubmit={v => console.log(v)}
         render={({ values }) => (
           <div>
-            <AccountFields manifestFields={sanitizedFields} t={t} />
+            <AccountFields
+              fillEncrypted={!!initialValues}
```

Comment:
`fillEncryted` if there is some values ?
Something like `hasValues` could be better, no ?

Reply:
Functionnaly, `AccountField` can fill empty encrypted value, so it seems more logic to me explicit the feature.